indy_common/config.py

```diff
@@ -103,3 +103,5 @@
 authPolicy = CONFIG_LEDGER_AUTH_POLICY
 
 SCHEMA_ATTRIBUTES_LIMIT = 125
+
+CONTEXT_SIZE_LIMIT = 131072
```

Comment:
We have a test that exceeds our limit for a context CONTEXT_SIZE_LIMIT, but does not seem to trigger the MSG_LEN_LIMIT during the Plenum check. Instead we get a PoolLedgerTimeoutException.  Is there compression of messages that would reduce the size of the message at the Plenum level?

The largest context we have seen so far is from schema.org at 121278 bytes. What do you recommend?

Reply:
>  Instead we get a PoolLedgerTimeoutException
Yes, I believe this is what expected from the client side if message is discarded because of `MSG_LEN_LIMIT ` checks.
`MSG_LEN_LIMIT ` checks are done on zstack level (network level) before parsing and handling request, so for the client it looks like a timeout.
Please note that `MSG_LEN_LIMIT ` checks are done before decoding the message (have a look at `_verifyAndAppend` in plenum's `zstack.py`).

I recommend just to increase `MSG_LEN_LIMIT ` in Plenum.